Library/Homebrew/cli/parser.rb

```diff
@@ -432,6 +442,70 @@ def hide_from_man_page!
 
       private
 
+      SYMBOL_TO_USAGE_MAPPING = {
+        text_or_regex: "<text>|`/`<regex>`/`",
+        url:           "<URL>",
+      }.freeze
+
+      def generate_usage_banner
+        command_names = ["`#{@command_name.tr("_", "-")}`"]
+        aliases_to_skip = %w[instal uninstal]
+        command_names += Commands::HOMEBREW_INTERNAL_COMMAND_ALIASES.map do |command_alias, command|
+          next if aliases_to_skip.include? command_alias
+
+          "`#{command_alias.tr("_", "-")}`" if command == @command_name
+        end.compact.sort
+
+        options = if @processed_options.any?
+          " [<options>]"
+        else
+          ""
+        end
```

Comment:
Would it be possible for it to show what the options are if there are only 2 or fewer? That way if there's only a few options it'll all be on the usage line, like `brew unlink [--dry-run] formula [formula]` rather than `brew unlink [options] formula [formula]`

Reply:
Yeah, that's probably doable. I'd point out that all commands have at least the four global options (`--debug`, `--quiet`, `--verbose`, `--help`) but those probably don't need to be in the usage string.

Would it make sense to do `brew unlink [--dry-run] [options] formula [...]` (or in the other order) or just nix the `[options]`?

Reply:
I think the manpage section for global options say they apply to all commands, so we can probably omit it in this case.

Reply:
Yeah, makes sense. I'll play around with the limit a little maybe and see what seems reasonable